internal/ambient/k8sclient.go

```diff
@@ -0,0 +1,74 @@
+package ambient
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// K8SClient represents a simple client for Kubernetes API.
+type K8SClient interface {
```

Comment:
@mcruzdev I think that dapr itself has a k8s util package that we can use to do this.. but if it is too complicated we can use this for now

Reply:
I saw that dapr has a util to create a kube client:
https://pkg.go.dev/github.com/dapr/dapr@v1.10.4/utils

But to create a configmap, I do not know.

Reply:
I can see that you are creating the configMap in the right way.